bigquery/README.md

```diff
@@ -43,7 +43,7 @@ PostgreSQL instance in Cloud SQL:
 - Enter an instance name and default password
 - Choose a region and zone or leave the defaults
 - Click on "configuration options", select "authorize networks", and add the
-  external IP of the VM that you noted earlier
+  external IP followed by /32 of the VM that you noted earlier.
```

Comment:
Is the /32 actually needed if you're entering a single IP?  I'm pretty sure I didn't enter it.

Reply:
I just checked it. You are correct, you don't need a to put a single IP in CIDR notation.